pkg/cmd/gtctl/root.go

```diff
@@ -50,11 +58,31 @@ func NewRootCommand() *cobra.Command {
 		"info log verbosity, higher value produces more output",
 	)
 
-	l := logger.New(os.Stdout, log.Level(flags.Verbosity), logger.WithColored())
-
 	// Add all top level subcommands.
 	cmd.AddCommand(version.NewVersionCommand(l))
 	cmd.AddCommand(cluster.NewClusterCommand(l))
 
 	return cmd
 }
+
+func runE(logger log.Logger, flags *flagArgs, _ *cobra.Command) error {
+	if err := maybeSetVerbosity(logger, log.Level(flags.Verbosity)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// maybeSetVerbosity will call logger.SetVerbosity(verbosity) if logger has a SetVerbosity method.
+func maybeSetVerbosity(logger log.Logger, verbosity log.Level) error {
+	type verboser interface {
+		SetVerbosity(log.Level)
+	}
+	v, ok := logger.(verboser)
+	if ok {
+		v.SetVerbosity(verbosity)
+		return nil
+	}
+
+	return fmt.Errorf("logger does not implement SetVerbosity")
```

Comment:
Question: If the log level not set, is there a default log level ?

Reply:
The default log level is [`0`](https://github.com/GreptimeTeam/gtctl/blob/develop/pkg/cmd/gtctl/root.go#L53), because the default varible value is 0.